Libplanet.Net.Tests/Transports/TransportTest.cs

```diff
@@ -140,45 +141,6 @@ public void AsPeer()
         }
 
         // This also tests ITransport.ReplyMessageAsync at the same time.
-        [SkippableFact(Timeout = Timeout)]
-        public async Task SendMessageAsync()
```

Comment:
Why is this test removed? Do we not want to test "normal" successful behavior?

Reply:
It wasn't intended deletion. I'll revert.

Reply:
Amended. (https://github.com/planetarium/libplanet/pull/2521/commits/bf7bba0bf2e0746e89885d2767cb0dff993bbb3b)